.github/workflows/ci.yaml

```diff
@@ -0,0 +1,44 @@
+name: Plugin CI
+on:
+  push:
+    branches: [ 'master' ]
+  pull_request:
+
+jobs:
+  tests:
+    name: "Tests ${{ matrix.php-version }} deps ${{ matrix.dependency-versions }}"
+    runs-on: ubuntu-22.04
+
+    strategy:
+      fail-fast: false
+      matrix:
+        # normal, highest, non-dev installs
+        php-version: [ '8.1', '8.2' ]
```

Comment:
php version should be equal 8.2 or higher.

Reply:
@Asisyas, done